src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs

```diff
@@ -491,19 +492,24 @@ protected override void ProcessRecord()
         {
             if (EffectiveParameterSet == MyParameterSet.RandomListItem)
             {
-                foreach (object item in InputObject)
+                // this allows for $null to be in an array passed to InputObject
+                foreach (object item in InputObject ?? _nullInArray)
                 {
-                    if (_numberOfProcessedListItems < Count) // (3)
+                    // (3)
```

Comment:
What is the comment for?

Reply:
Codefactor was complaining about the trailing comment so I moved it above.  If you look at the large comment above, it references (1), (2), (3) explaining the code so I left it.